roles/ee_builder/README.md

```diff
@@ -33,10 +32,8 @@ Order of preferences for images
     __ee_stream_images:
       upstream:
         base_image: quay.io/ansible/ansible-runner:latest
-        builder_image: quay.io/ansible/ansible-builder:latest
       downstream:
-        base_image: registry.redhat.io/ansible-automation-platform-23/ee-minimal-rhel8:latest
-        builder_image: registry.redhat.io/ansible-automation-platform-23/ansible-builder-rhel8:latest
+        base_image: registry.redhat.io/ansible-automation-platform-24/ee-minimal-rhel8:latest
```

Comment:
If my suggestion from the vars file about using RHEL9 is accepted then this will need to change as well to match.